docs/subscription_groups.md

```diff
@@ -0,0 +1,34 @@
+# Subscription groups
+
+Subscription groups are used to subscribe user to some marketing channels, for example sms or whatsapp.
+Subscription groups ids could be seen in Braze, please ask CRM team for an ID of the group that you need.
+
+## update user subscription to some group
+
+```php
+
+use App\Braze\User\Object\UserExternalId;
+use App\Braze\User\Object\UserSubscriptionGroupId;
+use Lingoda\BrazeBundle\Api\BrazeApiInterface;
+use Lingoda\BrazeBundle\Api\Constants\SubscriptionType;
+
+
+
+/**
+ * @throws HttpException
+ */
+public function updatePhoneSubscriptions(User $user, SubscriptionType $status): void
```

Comment:
❓ 
Is there a specific as to why the status is actually `SubscriptionType` object?